src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator start work");

```

Comment:
The message 'Excavator start work' is not informative enough. It should indicate more clearly that the Excavator is the type of Machine that is working. Consider rephrasing it to something like 'Excavator is starting its work' to make it more informative as per the checklist.